src/emitter.rs

```diff
@@ -254,7 +254,17 @@ impl<'a> YamlEmitter<'a> {
 /// * When the string looks like a number, such as integers (e.g. 2, 14, etc.), floats (e.g. 2.6, 14.9) and exponential numbers (e.g. 12e7, etc.) (otherwise, it would be treated as a numeric value);
 /// * When the string looks like a date (e.g. 2014-12-31) (otherwise it would be automatically converted into a Unix timestamp).
 fn need_quotes(string: &str) -> bool {
-    string.contains(|character: char| {
+    fn is_space(c: char) -> bool {
+        c == ' ' || c == '\t'
```

Comment:
No need to check for `'\t'` here because any string containing `'\t'` anywhere will be quoted.